contracts/DFTypes.sol

```diff
@@ -295,4 +295,84 @@ struct RevealProofArgs {
     uint256[2][2] _b;
     uint256[2] _c;
     uint256[9] _input;
-}
\ No newline at end of file
+}
+
+// THIS NEEDS TO BE UPDATED ANY TIME A CONSTANT IS ADDED
+struct AllConstants {
```

Comment:
@henryacaron the other option is to just return `arenaConstants, gameConstants, and snarkConstants`. Might be cleaner and then I just have to parse that on the graph side

Reply:
oh yeah this is super gnarly and non upgradeable, I think it makes more sense just to return all the constants from different structs and parse them like we already do for planets and players